panel/lxqtpanel.cpp

```diff
@@ -374,7 +372,7 @@ int LXQtPanel::getReserveDimension()
     return mHidable ? PANEL_HIDE_SIZE : qMax(PANEL_MINIMUM_SIZE, mPanelSize);
 }
 
-void LXQtPanel::setPanelGeometry(bool animate)
+void LXQtPanel::setPanelGeometry(bool animate, bool firstTime)
```

Comment:
Apart from what I said above, there's a problem with your last commit:

(1) Make an auto-hiding bottom panel with a 50-ms duration;
(2) Show it with the cursor, reconfigure it and change the duration to 500 ms.
(3) close the config dialog and you'll see a weird effect.

I assure you that I encountered so many unexpected problems in the case of this PR and, at last, found a short and *safe* way to fix them. You even made it shorter with your first suggestion, so I'll update the PR by using it.